samples/cellular/modem_shell/README.rst

```diff
@@ -122,6 +122,10 @@ in a separate plain AT command mode where also pipelining of AT commands is supp
    When using ``at`` command, any quotation marks (``"``), apostrophes (``'``) and backslashes (``\``) within the AT command syntax must be escaped with a backslash (``\``).
    The percentage sign (``%``) is often needed and can be written as is.
 
+.. note::
+   In order to support provisioning certificates using the AT%CMNG command, occurences of escaped newlines ``\\n`` are replaced by ``\r\n`` internally.
+   To use this feature, remove existing occurences of ``\r`` and replace ``\n`` with ``\\n`` in the AT command string.
```

Comment:
```suggestion
   In order to support provisioning certificates using the ``AT%CMNG`` command, occurrences of escaped newlines ``\\n`` are replaced by ``\r\n`` internally.
   To use this feature, remove existing occurrences of ``\r`` and replace ``\n`` with ``\\n`` in the AT command string.
```

Reply:
I'm not sure combining the notes is good. They are all for very different purpose. And the new note is not generally about ``at`` shell command but only ``at AT%CMNG``.